docs/metadata.rst

```diff
@@ -124,56 +124,27 @@ explanatory purposes.
           "version": 1 // NatSpec version
         }
       },
-      // Required: Compiler settings. Reflects the settings in the JSON input during compilation.
-      // Check the documentation of standard JSON input's "settings" field
+      // Required: Compiler settings.
+      // Reflects the settings in the JSON input during compilation, except the
+      // "compilationTarget" and the "libraries" fields. Check each field for details.
+      // See the standard JSON input's "settings" field docs for the rest.
       "settings": {
         // Required for Solidity: File path and the name of the contract or library this
-        // metadata is created for.
+        // metadata is created for. This field is not present in the standard JSON input settings.
         "compilationTarget": {
           "myDirectory/myFile.sol": "MyContract"
         },
-        // Required for Solidity.
-        "evmVersion": "london",
-        // Optional: Only present if not null.
-        "eofVersion": 1,
         // Required for Solidity: Addresses for libraries used.
+        // Note that metadata has a different format for "libraries" field than the standard JSON input.
+        // metadata format = { "MyLib.sol:MyLib": "0x123123..." }
+        // standard JSON input format = { "MyLib.sol": { "MyLib": "0x123123..." } }
         "libraries": {
-          "MyLib": "0x123123..."
+          "MyLib.sol:MyLib": "0x123123..."
         },
-        "metadata": {
-          // Reflects the setting used in the input json, defaults to "true"
-          "appendCBOR": true,
-          // Reflects the setting used in the input json, defaults to "ipfs"
-          "bytecodeHash": "ipfs",
-          // Reflects the setting used in the input json, defaults to "false"
-          "useLiteralContent": true
-        },
-        // Optional: Optimizer settings. The fields "enabled" and "runs" are deprecated
-        // and are only given for backward-compatibility.
-        "optimizer": {
-          "details": {
-            "constantOptimizer": false,
-            "cse": false,
-            "deduplicate": false,
-            // inliner defaults to "false"
-            "inliner": false,
-            // jumpdestRemover defaults to "true"
-            "jumpdestRemover": true,
-            "orderLiterals": false,
-            // peephole defaults to "true"
-            "peephole": true,
-            "yul": true,
-            // Optional: Only present if "yul" is "true"
-            "yulDetails": {
-              "optimizerSteps": "dhfoDgvulfnTUtnIf...",
-              "stackAllocation": false
-            }
-          },
-          "enabled": true,
-          "runs": 500
-        },
-        // Required for Solidity: Sorted list of import remappings.
-        "remappings": [ ":g=/dir" ]
```

Comment:
Just noticed this and this is also inaccurate so we should fix it (we still have the same comment in input docs).

The compiler does not sort remappings before putting them in metadata. In the input they are not expected to be sorted either. The order can matter in corner cases. For example if prefixes are identical, the one specified last wins (see [Import Remapping](https://docs.soliditylang.org/en/develop/path-resolution.html#import-remapping)).

Reply:
Can you do this in this PR?

Reply:
Updated from 

```
// Optional: Sorted list of remappings
```

to
```
// Optional: List of remappings
```

Hope I got it right :)

Reply:
Yeah, looks good. Thanks!